custom/abt/reports/fixture_utils.py

```diff
@@ -231,7 +231,7 @@ def fixture_data_item_to_dict(
 
     """
     return {
-        key: field_list.field_list[0].field_value
+        key: field_list.field_list[0].field_value if field_list.field_list else None
```

Comment:
is this preferable to skipping the row?

Reply:
I wasn't sure, but I chose this approach because the row isn't completely empty: It has an ID. 

In general I figured this would would allow us to deal with rows where some values are there and others aren't. But in the context of VectorLink, this is not valid location, and the fixture is used by apps as well as the report filters. So the row has been deleted.

This change should prevent the report from breaking when it's building the location filters, and I'm hoping it will allow admins to notice an empty location, and fix the fixture, so that app users can't select it.